src/lib/webcontainer.ts

```diff
@@ -280,6 +281,24 @@ function does_file_exist(files: FileSystemTree, path: `./${string}`) {
 	return !!subtree[file];
 }
 
+async function read_file(path: string): Promise<string>;
+async function read_file(path: string, as_string: false): Promise<Uint8Array>;
```

Comment:
What's this funky business? 👀

Reply:
This is an overload function in typescript...it's just to avoid having to check everywhere we already used read_file that the returned value is a string. Using overloads we can be sure that if you are passing an empty as_string you get a string back (given it default to true) and we don't have to refactor everywhere we use read_file